package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-ui-components",
-  "version": "3.0.0",
+  "version": "4.0.0-beta.5",
```

Comment:
Should this be reset to 3.0.0 before merging to dev or is it better to leave as an indicator that betas were released?

Reply:
I think we can reset it back to `3.0.0` and then update it to `4.0.0` when making the actual stable release.

We have the beta-release commits and tags available, so we can utilise that information.

Reply:
I wonder what git-flow says about this. I kind of think that development should reflect what has been released as beta but not sure about it. It would be more clear what beta version should be released next with a new feature branch and having the beta version in package json would help with that. It should not conflict with hotfixes either since those should be merged to develop as well.

Reply:
I propose we leave it as it is. This way it shows history better and it's clear what has been published.